src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -482,8 +482,12 @@ protected function denormalizeParameter(\ReflectionClass $class, \ReflectionPara
      *
      * @internal
```

Comment:
wait, I completely missed that this was internal
we can add the argument 3.4 now then!

Reply:
is it that important though that we should add the deprecation in 3.4? i guess people jumping from 3 to 5 without going through 4 will have more of a problem.

and afaik api platform is using these things. or is it not @joelwurtz

Reply:
Yes it is: https://github.com/api-platform/core/blob/master/src/Serializer/AbstractItemNormalizer.php#L551

Reply:
@dbu we cannot add new deprecations in 3.4. Deprecations are always added in master.

Reply:
thats what i was assuming too, hence this pull request goes against master.

so is there anything to change here or is it fine as is?

Reply:
Thanks for the link @joelwurtz - this should be reported as a bug in api-platform: not using `@internal` classes/methods is not an option. ping @dunglas FYI

Reply:
I did it on purpose (IIRC I created both codes at the same time). We’ll change it in API Platform too no worry.